nodejs/pulumi/package-lock.json

```diff
@@ -0,0 +1,593 @@
+{
+  "name": "pulumi",
+  "version": "1.0.0",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "pulumi",
+      "version": "1.0.0",
+      "license": "ISC",
```

Comment:
@jkodroff I know that the default license type is `ISC` but could you change to `Apache-2.0`? The whole dagger/examples repo is under the `Apache-2.0` license and keeping this consistent will mean that we don't need to change https://github.com/dagger/examples/blob/main/LICENSE

I know this will get re-generated after you change the `package.json` with similar comment.

Reply:
Fixed.